logaccess/loki/build.gradle.kts

```diff
@@ -41,6 +41,7 @@ dependencies {
     implementation(libs.ktorKotlinxSerialization)
 
     testImplementation(testFixtures(projects.logaccess.logaccessSpi))
+    testImplementation(projects.utils.logging)
```

Comment:
Why is this required here now if no code changes are done in this module?

Reply:
It's not, I have removed it.